runner/runner.go

```diff
@@ -53,25 +53,23 @@ func NewCommand(argv []string, env map[string]string, timeout time.Duration, sna
 	return &Command{Argv: argv, EnvVars: env, Timeout: timeout, SnapshotId: snapshotId}
 }
 
-type Runner interface {
+// Controller lets clients Control commands.
+type Controller interface {
```

Comment:
The name hierarchy could be clearer, maybe something like:
```
Runner {
  CmdController {
    StatusRecorder
    CmdExecerAdapter {
      Execer
    }
  }
  StatusChecker
}
```
This isn't great either but it's clearer I think.